release-notes/opensearch-security.release-notes-2.11.0.0.md

```diff
@@ -0,0 +1,40 @@
+## 2023-10-18 Version 2.11.0.0
+
+Compatible with OpenSearch 2.11.0
+
+### Enhancements
+* Authorization in Rest Layer ([#2753](https://github.com/opensearch-project/security/pull/2753))
+* Improve serialization speeds ([#2802](https://github.com/opensearch-project/security/pull/2802))
+* Integration tests framework ([#3388](https://github.com/opensearch-project/security/pull/3388))
+* Allow for automatic merging of dependabot changes after checks pass ([#3409](https://github.com/opensearch-project/security/pull/3409))
+* Support security config updates on the REST API using permission([#3264](https://github.com/opensearch-project/security/pull/3264))
+* Expanding Authentication with SecurityRequest Abstraction ([#3430](https://github.com/opensearch-project/security/pull/3430))
+* Add early rejection from RestHandler for unauthorized requests ([#3418](https://github.com/opensearch-project/security/pull/3418))
+
+### Bug Fixes
+* Removing opendistro-job-scheduler-lock from list of protected indices ([#3383](https://github.com/opensearch-project/security/pull/3383))
```

Comment:
This was included in 2.10: https://github.com/opensearch-project/security/pull/3351